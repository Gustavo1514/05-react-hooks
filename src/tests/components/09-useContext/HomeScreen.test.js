const { shallow, mount } = require('enzyme')
const {
  default: HomeScreen
} = require('../../../components/09-useContext/HomeScreen')
const { UserContext } = require('../../../components/09-useContext/UserContext')

describe('Pruebas en <HomeScreen/>', () => {
  const user = {
    name: 'Henry',
    email: 'henry@gmail.com'
  }
  const wrapper = mount(
    <UserContext.Provider
      value={{
        user
      }}
    >
      <HomeScreen />
    </UserContext.Provider>
  )
  test('debe renderizar correctamente el componente', () => {
      expect(wrapper).toMatchSnapshot()
  })
})
