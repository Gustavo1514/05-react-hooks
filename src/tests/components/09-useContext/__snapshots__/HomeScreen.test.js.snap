// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScreen/> debe renderizar correctamente el componente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Henry",
   "email": "henry@gmail.com"
}
  </pre>
</div>
`;
